src/main/java/org/isf/patient/gui/SelectPatient.java

```diff
@@ -616,21 +621,17 @@ public int getColumnCount() {
 
 		@Override
 		public Object getValueAt(int r, int c) {
-			Patient patient = patSearch.get(r); 
+			Patient patient = patSearch.get(r);
 			if (c == -1) {
 				return patient;
 			} else if (c == 0) {
 				return patient.getCode();
-			}  else if (c == 1) {
+			} else if (c == 1) {
 				return patient.getName();
-			} /*else if (c == 2) {
-				return patient.getAge();
-			} else if (c == 3) {
-				return patient.getSex();
-			} else if (c == 4) {
-				return patient.getCity() + " "
-						+ patient.getAddress();
-			} */
+			} /*
```

Comment:
why not just get rid of the commented out code; it is hard to read anyway

Reply:
the commented code was already there, thanks!